### PR TITLE
Supports Chinese file name

### PR DESCRIPTION
**Pull request by xiaojian-hong**
Allow Chinese characters in `isFileName` function.

### CODE REVIEW

**Comment by yumauri** [comment]
Wow, thank you!

Did you try to send files with Chinese characters to Gotenberg? Does it work?

Also, maybe it will be better to use `/^[\p{L}\w\s\(\),-]+\.[A-Za-z0-9]+$/u`? To match any characters in any language possible.

File: src/internal/source-checkers.ts [hunk 1]
```diff
@@ -82,7 +82,7 @@ export const isIterable = (
  * Check, if given argument is simple string, and presumably is is just file name
  * I hope no one will wants to use filename longer than 50 symbols :)
  */
-const fileNameRE = /^[\w\s\(\),-]+\.[A-Za-z0-9]+$/
+const fileNameRE = /^[\u4e00-\u9fa5\w\s\(\),-]+\.[A-Za-z0-9]+$/
```
Comment (yumauri) [inline src/internal/source-checkers.ts:85]:
```suggestion
const fileNameRE = /^[\p{L}\w\s\(\),-]+\.[A-Za-z0-9]+$/u
```

**Comment by xiaojian-hong** [comment]
> Wow, thank you!
> 
> Did you try to send files with Chinese characters to Gotenberg? Does it work?
> 
> Also, maybe it will be better to use `/^[\p{L}\w\s\(\),-]+\.[A-Za-z0-9]+$/u`? To match any characters in any language possible.

Yes,  I tested files with Chinese characters to Gotenberg, it works.
Your suggestion `/^[\p{L}\w\s\(\),-]+\.[A-Za-z0-9]+$/u` is even better.

**Comment by xiaojian-hong** [comment]
Seems there is an issue in ESLint for unicode regexps. https://github.com/buzinas/tslint-eslint-rules/issues/289
`yarn lint` throws  `ERROR: 85:20  no-empty-character-class  don't use empty classes in regular expressions`.
I just disabled the tslint rule `no-empty-character-class` in this regexp.

**Comment by yumauri** [comment]
I doubt TSLint will fix this, because it has been deprecated, so until I move to ESLint, I'm OK with this :) Thank you!

**Comment by yumauri** [comment]
Released in version [0.6.2](https://github.com/yumauri/gotenberg-js-client/releases/tag/v0.6.2) 🎉

**Pull request merged**